### ISSUE TITLE
the shapes of input and output of glu function aret not consistent

### ISSUE THREAD

**Issue by softrime** [Kyubyong/deepvoice3#17]
`def glu(inputs):`
`    '''Gated linear unit`
`    Args:`
`      inputs: A tensor of even dimensions. (N, Tx, 2c)`
`    Returns:`
`      outputs: A tensor of the same shape and dtype as inputs.`
`    '''`
`    a, b = tf.split(inputs, 2, -1)  # (N, Tx, c) * 2`
`    outputs = a * tf.nn.sigmoid(b)`
`return outputs`
input shape is (N, Tx, 2c) while output shape should be (N, Tx, c) 
I'm confused about consistent shapes of them.

**Comment by softrime** [comment]
Oh I got what this mean. consistent shapes should mean input and output in conv block
